### PR TITLE
docs: Toasts option maxVisible

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

I forgot to document the option to display a maximum number of visible toasts which was provided in PR  #458.

# Changes

- Documention option in showcase.
- Update screenshots (more documentation pushes a bit the content in the screenshots).

### CODE REVIEW

File: e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-darwin.png
Comment (dskloetd) [inline e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-darwin.png:1]: This doesn't belong in this PR, right?

Comment (peterpeterparker) [reply e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-darwin.png:1]: Correct. I moved it to #462.

Comment (dskloetd) [reply e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-darwin.png:1]: I still see 19 files changed in this PR.

Comment (peterpeterparker) [reply e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-darwin.png:1]: I reverted the "local" one. The other screenshots remaining changes are expected.

File: e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-linux.png
Comment (dskloetd) [inline e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-linux.png:1]: The only change I see is the line height in the icon description. How is that a result of this PR?

Comment (peterpeterparker) [reply e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-linux.png:1]: As you can see, if I revert the snapshots the tests are failing.

If I debug the issue locally, those are the differences.

Toast-warn-1-diff
Toast-success-1-diff
Toast-multiline-message-1-diff
Toast-message-1-diff
Toast-info-1-diff
Should-display-multiple-toasts-for-the-top-position-1-diff
Should-display-multiple-to-6c234-user-is-able-to-close-one-1-diff
Should-close-toast-1-diff
Toast-error-1-diff

Comment (dskloetd) [reply e2e/toasts.e2e.ts-snapshots/Should-close-toast-1-Google-Chrome-linux.png:1]: Strange. Thanks for trying.

**Review by dskloetd — APPROVED** [review]
